Apps/Inc/Temperature.h

```diff
@@ -22,6 +22,8 @@
 #define AUX_I2C_BLANK 0
 #define AUX_I2C_NO_TRANSMIT 0x7
 
+#define TEMP_ERR_OUT_BOUNDS 42424242
```

Comment:
Where did 42424242 come from?

Reply:
Its an impossibly high temperature (42424.242 degrees C) that we return as an error code if we get a voltage reading from the temperature sensor that is out of bounds of what the datasheet says the LMT87 will return (eg. 5 Volts). 

The motivation for this is:
1. The temperature sensors will never actually measure ~42,000 C
2. This number is high enough to cause an overvoltage fault
3. When printed out during debugging, it looks made up to a human